rust_src/src/base64.rs

```diff
@@ -2,6 +2,7 @@
 use std::{cmp::max, slice};
 
 use libc::{c_char, c_uchar};
+use line_wrap::LineEnding;
```

Comment:
Minor style nit. Please leave the blank line so `use remac_macros` is on a line by itself.

Reply:
Roger that :+1: